src/core/initialize.rs

```diff
@@ -96,15 +96,15 @@ impl StackTraceGetter {
                                     }
                                     // Windows
                                     std::io::ErrorKind::PermissionDenied => {
-                                        return Err(MemoryCopyError::ProcessEnded)
+                                        return Err(MemoryCopyError::ProcessNotLocked)
```

Comment:
I'm speculating that this issue may also be present on windows, either way I think ProcessEnded is not a good error to provide in this case, and ProcessNotLocked is more descriptive.

Reply:
I'll test this before merging. I recall that process exit on Windows can result in an EPERM-like error code, but it might happen when we're walking through VM memory and not here.